Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYApplePayPaymentProvider.m

```diff
@@ -151,20 +156,29 @@ - (BOOL)isAvailable
 	// checks if the client is setup to use Apple Pay
 	// checks if device hardware is capable of using Apple Pay
 	// checks if the device has a payment card setup
+	
+	if (SYSTEM_VERSION_LESS_THAN(@"10.0")) {
```

Comment:
you should check for class availability, not version number.

this should work `    Class PaymentClass = ([PKPaymentAuthorizationController class]) ?: [PKPaymentAuthorizationViewController class];
`